src/release/breaking-changes/wide-gamut-framework.md

```diff
@@ -0,0 +1,135 @@
+---
+title: Migration guide for wide gamut Color
+description: Description on Color changes for wide gamut support and migration
+             instructions.
+---
+
+## Summary
+
+The API for the [`Color`][] class in `dart:ui` is changing to support wide gamut
+colors.
+
+## Context
+
+The Flutter engine [already supports wide gamut color][] with [Impeller][], and the support is now being added [to the framework][].
+
+The iOS devices Flutter targets support rendering a larger array of colors,
+specifically the [DisplayP3][] color space. After this change, the Flutter
+framework will be able to render all of those colors on iOS Impeller and `Color`
+will be better prepared for the future addition of color spaces or changes to
+color component bit depth.
+
+## Description of change
+
+Changes to [`Color`][]:
+
+ 1. Adds a enum field which will specify its [`ColorSpace`][].
+ 1. Adds API to use normalized floating-point color components.
+ 1. Removes API that uses 8-bit unsigned integer color components that can lead
+    to data loss.
+
+Changes to [`ColorSpace`][]:
+
+ 1. `displayP3` will be added.
+
+## Migration guide
+
+### 8-bit unsigned integer constructors
+
+Constructors like `Color.fromARGB` will remain unchanged and have continued
+support. If one wants to take advantage of Display P3 colors they will have to
+use the new `Color.from` constructor that takes normalized floating-point
+color components.
+
+```dart
+// Before
+final magenta = Color.fromARGB(0xff, 0xff, 0x0, 0xff);
+// After
+final magenta = Color.from(alpha: 1.0, red: 1.0, green: 0.0, blue: 1.0)
+```
+
+### Implementors of `Color`
+
+There are new methods being added to `Color` so any class that `implements`
+Color will break and have to implement the new methods like `Color.a`, and
+`Color.b`. Ultimately implementors should be migrated to take advantage of the
+new API. In the short-term, these methods can easily be implemented without
+changing the underlying structure of your class.
+
+For example:
+
+```dart
+class Foo implements Color {
+  int _red;
+
+  @override
+  double get r => _red * 255.0;
+}
+```
+
+{{site.alert.note}}
+  The hope is to eventually make Color `sealed`. Now might be a good opportunity
+  to stop implementing `Color`. Instead of inheritance, switch to composition.
+{{site.alert.end}}
```

Comment:
```suggestion
:::note
We plan to eventually lock the `Color` class down and make it `sealed`. Now might be a good opportunity to switch from [inheritance to composition][] and stop reimplementing `Color`.
:::

[inheritance to composition]: https://en.wikipedia.org/wiki/Composition_over_inheritance
```

Reply:
I don't see another instance of the `:::note` syntax anywhere in the repo.  Are you sure it's available here?

Reply:
Yes, I am 100% sure.

Reply:
done